.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2.1
+
+orbs:
+  orb-tools: circleci/orb-tools@9.0.0
+
+workflows:
+  validate_publish-dev:
+    jobs:
+      - orb-tools/lint
+      - orb-tools/pack
+      - orb-tools/publish-dev:
```

Comment:
Don't we want to limit publishing to dev to the `develop` branch? That makes it easy to see what is currently deployed there.

Reply:
We could, but then you cannot test it until it's merged to `develop`.

Reply:
What I could do is publish a SHA version on every push and only publish a dev version on `develop`.

Reply:
Yes, sounds good :+1: